bookish/controllers/bookish.py

```diff
@@ -1,7 +1,17 @@
 from flask import request
 from bookish.models.example import Example
+from bookish.models.books import Books
+from bookish.models.users import Users
+from bookish.models.checked_out_books import Checked_Out_Books
 from bookish.models import db
 
+import hashlib
+
+
+def hashing_algorithm(string):
+    result = hashlib.md5(string.encode('utf-8')).hexdigest()
+    return result
```

Comment:
Also this isn't a `controller` and it shouldn't be in the controller folder 😄 You can create a `helper` folder/file for auxiliary functions.